react/src/App.tsx

```diff
@@ -1,9 +1,17 @@
+import { IconButton } from '@material-ui/core';
+import { Close } from '@material-ui/icons';
+import { SnackbarKey, SnackbarProvider } from 'notistack';
```

Comment:
These should come after the React import!

Reply:
There's already a `@material-ui/core` import in this file. Could you combine them please?